feature-libs/asm/styles/components/_asm-save-cart-dialog.component.scss

```diff
@@ -0,0 +1,214 @@
+$sapFiori_Button_Standard_TextColor: #1d2d3e;
```

Comment:
May you tell us the source of the scss(copied from xxx and modified sth to let us review easier?

Reply:
copied from `_asm-bind-cart-dialog.component.scss`